### PR TITLE
fix PrimaryKey unsupported GUID type issue

### PR DESCRIPTION
**Pull request by serenabenny**

### CODE REVIEW

**Comment by TimGeyssens** [comment]
Thanks for the pull request, I'll review asap!

File: src/UIOMatic/Controllers/PetaPocoObjectController.cs [hunk 1]
```diff
@@ -357,15 +366,39 @@ public object PostCreate(ExpandoObject objectToCreate)
 
             foreach (var prop in objectToCreate)
             {
+                var propKey = prop.Key;
+                PropertyInfo propI = currentType.GetProperty(propKey);
+                //object value
+                if (propI.GetCustomAttributes().Any(x => x.GetType() == typeof(UIOMaticIgnoreFieldAttribute)))
+                {
+                    switch (propI.Name.ToLower())
```
Comment (TimGeyssens) [inline src/UIOMatic/Controllers/PetaPocoObjectController.cs]: what's the purpose of this switch? Seems like it's related to a specific project implementation?

**Comment by TimGeyssens** [comment]
Also why the changes to the files

src/UIOMatic/UIOMatic.csproj
src/UIOMatic/Web.Debug.config
src/UIOMatic/Web.Release.config
src/UIOMatic/Web.config
src/UIOMatic/Properties/AssemblyInfo.cs

**Comment by serenabenny** [comment]
Sorry,Because my environment is to use Umbraco 7.2.8 , you can ignore this changes.

**Comment by TimGeyssens** [comment]
Ok, could you also take a look at the other comment? Thanks :)

**Comment by TimGeyssens** [comment]
There seems to be some other stuff in the pull request so I just picked the changes related to the PrimaryKey unsupported GUID type issue and those will be part of the next release! Again thanks for the code :)

**Pull request closed without merging**